<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HooKAllMsgBoxAppClass</class>
 <widget class="QWidget" name="HooKAllMsgBoxAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HooKAllMsgBoxApp</string>
  </property>
  <widget class="QTextEdit" name="m_TextEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>511</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnStart">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>startHook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnstop">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>stophook</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btnMsg">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>MsgBox</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hookallmsgboxapp.qrc"/>
 </resources>
 <connections/>
</ui>
